include/mqt-core/dd/Operations.hpp

```diff
@@ -101,8 +101,8 @@ MatrixDD getInverseDD(const qc::Operation& op, Package& dd,
  * @brief Apply a unitary operation to a given vector DD.
  *
  * @details This is a convenience function that realizes @p op times @p in and
- * correctly accounts for the permutation of the operation's qubits as well as
- * automatically handles reference counting.
```

Comment:
I think I would personally prefer to keep the old wording in most of the places where this was changed. Essentially, we are still doing some kind of reference counting, but only at the top level.